Common/include/option_structure.hpp

```diff
@@ -1056,18 +1045,98 @@ inline SST_ParsedOptions ParseSSTOptions(const SST_OPTIONS *SST_Options, unsigne
   return SSTParsedOptions;
 }
 
+/*!
+ * \brief SA Options
+ */
+enum class SA_OPTIONS {
+  NONE,     /*!< \brief No option / default. */
+  NEG,      /*!< \brief Negative SA. */
+  EDW,      /*!< \brief Edwards version. */
+  FT2,      /*!< \brief Use FT2 term. */
```

Comment:
Same problem as with "modified" "unmodified" if we want to default to noft2 we need two options, and one will be redundant

Reply:
But we can change to WITHFT2 to avoid confusion

Reply:
For compatibility with previous config files and avoid problems with transition, see https://github.com/su2code/SU2/pull/1066#discussion_r472827852, I think it is better to set the default to the no-ft2 version and add activate it with WITHFT2.

Reply:
I think the nasa website should be leading here. If they call it SA-NOFT2, then I think for the end user it is most clear if we use SA_OPTIONS=NOFT2
Also, if the user sees in the config file that the turbulence model is SA, then it should be SA, and not SA-neg, or SA-noft2.
We should prevent submodels being activated in a hidden way. Maybe it is even better to either force the user to provide an SA_OPTION, to make her aware of the change (my preference), or give a warning that no SA_OPTION is found (might be missed).

Reply:
I would agree with you Nijso, if we were doing it from scratch, but that boat sailed and we are not going to break backward compatibility like that.
We can think about it for V8.

But I have a good solution for the "legal" combinations problem, to use the "illegal" ones it will be necessary to put "EXPERIMENTAL"  in the SA options.
Ok?

Reply:
Great, done!